packages/autocannon-ui-frontend/src/components/App.jsx

```diff
@@ -110,7 +110,10 @@ function App() {
           {results.map((resultSet, index) => (
             <Grid item key={index} className={classes.resultsGridItem}>
               <ResultSet
-                data={resultSet}
+                data={{
+                  ...resultSet,
+                  index
```

Comment:
What if you use another property to do this?

Reply:
You mean to pass the index property as another one, like:
ResultSet
data = resultSet
resultIndex = index ?

Reply:
yep

Reply:
Done